ExternalRevisions/Vulkan-Headers_repo_revision

```diff
@@ -1 +1 @@
-fc6c06ac529e4b4b6e34c17cc650a8f62dee2eb0
+5a895c363dd689fdd58a4d336cceda77eca3a62f
```

Comment:
The main issue is that until the spec PR is not approved and merged, Vulkan-Headers cannot be updated with the xml data that contains the extension. Once the spec PR is merged then I will be able to start merging all other repositories that rely on Vulkan-Headers (which are VVL, CTS and MoltenVK). It's pretty much a domino effect, but as mentioned until the spec PR is approved and merged, the only thing I can do is update the Vulkan-Headers repository this change is pointing to so it is based on top of current MoltenVK's hash with the patch applied on top. If there's anything else I could do to facilitate pulling this change do let me know. Otherwise, I'll just update this PR once MoltenVK pulls new headers so there are not conflicts.